pylint/utils/utils.py

```diff
@@ -17,12 +17,32 @@
 import sys
 import textwrap
 import tokenize
+from typing import TYPE_CHECKING, Any, List, Optional, Pattern, Tuple, Union, overload
 
 from astroid import Module, modutils
 
 from pylint.constants import PY_EXTS
 
+if TYPE_CHECKING:
+    from typing import Literal
+
+    from pylint.checkers.base_checker import BaseChecker
+
 DEFAULT_LINE_LENGTH = 79
+GLOBAL_OPTION_BOOL = Union[
+    "Literal['ignore-mixin-members']",
+    "Literal['suggestion-mode']",
+    "Literal['analyse-fallback-blocks']",
+    "Literal['allow-global-unused-variables']",
+]
```

Comment:
No need to use `Union` here. You can pass multiple values to `Literal`. Same for the other type aliases.
https://docs.python.org/3/library/typing.html#typing.Literal

Reply:
I found that for some reason `VAR = "Literal['string']"` doesn't work. But with 
```if sys.version_info >= (3, 8):
    from typing import Literal
else:
    from typing_extensions import Literal
```
this is again possible!